poetry/console/commands/init.py

```diff
@@ -263,7 +263,8 @@ def _find_best_version_for_package(
         if not package:
             # TODO: find similar
             raise ValueError(
-                "Could not find a matching version of package {}".format(name)
+                "Could not find a matching stable version of package {}. "
+                "Use --allow-prereleases to remove restrictions.".format(name)
```

Comment:
Not that I am a maintaner or sth, but I think this message should depend on the value of `allow_prereleases` argument. Now it would advice user to use  `--allow-prereleases` even if they already used it. 🤷‍♂